### ISSUE TITLE
Path/Line should be focusable

### ISSUE THREAD

**Issue by Umer-Hayyat** [anseki/leader-line#59]
Hi, 
Path/Line should be focusable in order to support keyboard accessibility.

**Comment by anseki** [comment]
Hi @Umer-Hayyat, thank you for your comment.

I think that is not good idea.
The lines should never disturb UI even if the line is overlapped on the contents.
And also, the lines never be UI.
If you want something for UI, use other elements that is focusable and control the lines.

**Comment by anseki** [comment]
No reply came, then I close this abandoned issue.

**Comment by Umer-Hayyat** [comment]
you can close @anseki i am no more using this as it does not fulfil my requirements

**Comment by anseki** [comment]
It's OK. :smile: 
This was already closed because you did not reply.
